rotkehlchen/types.py

```diff
@@ -556,6 +556,8 @@ class EnsMapping(NamedTuple):
 class CostBasisMethod(SerializableEnumMixin):
     FIFO = 1
```

Comment:
since those are not put in the DB so changing them is fine, can you use auto() everywhere?